.github/workflows/beam_PreCommit_Java.yml

```diff
@@ -155,7 +155,7 @@ env:
 jobs:
   beam_PreCommit_Java:
     name: ${{ matrix.job_name }} (${{ matrix.job_phrase }})
-    timeout-minutes: 180
+    timeout-minutes: 360
```

Comment:
```suggestion
```

Actually, we can probably just cut this line - 360 is the default - https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes